app/controllers/devise_token_auth/confirmations_controller.rb

```diff
@@ -21,10 +21,12 @@ def show
         yield @resource if block_given?
 
         redirect_to(@resource.build_auth_url(params[:redirect_url], {
-          token:                        token,
-          client_id:                    client_id,
+          "access-token":               token,
```

Comment:
Should we maybe using the values here:

https://github.com/lynndylanhurley/devise_token_auth/blob/536911c7d3f178637423b77fe1179388ddf62b9e/lib/devise_token_auth/engine.rb#L38

Like they're used here?
https://github.com/lynndylanhurley/devise_token_auth/blob/ae87cae3a24869a5aceab5addc3f3c1041bf39d3/app/models/devise_token_auth/concerns/user.rb#L222

Just an idea.

Reply:
Are you referring to the hash key style? If so, they _should_ behave the same way: 

`{:"access-token" => "foo"} == {"access-token": "foo"} # true` 

Also, to use the former, we'd need to convert the rest of the hash keys to the hashrocket style.

Reply:
No, I guess I just mean that maybe we should use the configured keys:

`DeviseTokenAuth.headers_names[:"access-token"]`

rather than assume the user is using` access-token`. DeviseTokenAuth.headers_names seems to allow you to override the header names. I don't really know how many people are using that feature, probably not a lot because it's not used across the library.

Reply:
Ah, sorry - it's still early in EST. 😛 

I think that's a good idea. What would you propose we do in the case where keys aren't present in `DeviseTokenAuth.headers_names`, like `client_id`? Should those be added to `headers_names` in lib/devise_token_auth/engine.rb, even though they might be phased out at some point? (As I'd probably argue they should.)

Reply:
I would say things that are going to be phased out possibly in the future shouldn't get this treatment - I'd be okay with you only doing this with the essential keys, basically those already in DeviseTokenAuth.headers_names.

@MaicolBen perhaps you have an opinion on this too?